test/cases/compile_errors/deprecated.zig

```diff
@@ -0,0 +1,10 @@
+const bad = @deprecated(42);
+
+pub export fn foo() usize {
+    return bad;
+}
+
+// error
+// target=native
+//
+// ":1:13: error: found deprecated code",
```

Comment:
```suggestion
// ":1:13: error: found deprecated code"
```

Reply:
Also, remove the `target=native` line; it's redundant (since it's the default), I'm trying to get people to stop writing it :P